custom/icds_reports/utils/aggregation_helpers/agg_awc.py

```diff
@@ -315,7 +315,7 @@ def updates(self):
         FROM (
             SELECT DISTINCT(awc_id)
             FROM agg_awc
-            WHERE month = %(prev_month)s AND usage_num_hh_reg > 0 AND awc_id <> 'ALL'
+            WHERE month <= %(prev_month)s AND usage_num_hh_reg > 0 AND awc_id <> 'ALL'
```

Comment:
this should read `WHERE month = %(prev_month)s AND num_launched_awcs > 0 AND aggregation_level=5`
The last one lets postgres use table constraints on the child tables to only scan the relevant ones. And the first bit fixes the logic. We do not want to go all the way back and count forms forever, just see if it was launched last month as per https://github.com/dimagi/commcare-hq/pull/22056